### DISCUSSION TITLE
Is it possible to ensure Alpine components run properly when two versions of Alpine are loaded?

### DISCUSSION THREAD

**Discussion by alexstandiford** [alpinejs/alpine#2065] [1. Help] [answered]
AlpineJS is starting to grow in popularity, and I'm starting to notice that it's popping up in WordPress plugins and themes. I'm concerned that if multiple plugins are using AlpineJS that there may be a collision between the two plugins.

If there isn't a way to-do this now, I'm wondering if it would make sense to make it possible to replace the `x-` with a custom prefix, something like `custom-x-` instead. Perhaps that can tie the correct instance of Alpine to the components that intend to use that instance.

I imagine this could be added as an argument to `init`

**Comment by SimoTod** [comment, marked as answer]
It's already possible in v3. Assuming you are bundling your plugin

```html
import Alpine from 'alpinejs' 

... 

Alpine.prefix('myplugin-x-')
Alpine.start()
```

It will be down to whoever writes the plugin to use it.

**Reply by alexstandiford** [reply]
Sweet. I had hoped this was already a thing but couldn't find any info on it. Thanks for sharing!
